Source/DafnyLanguageServer/DafnyLanguageServer.cs

```diff
@@ -11,9 +12,11 @@
 using OmniSharp.Extensions.LanguageServer.Server;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
 
 namespace Microsoft.Dafny.LanguageServer {
   public static class DafnyLanguageServer {
+    public static IReadOnlyList<string> LoadErrors = new List<string>();
```

Comment:
Just to make sure that you didn't miss a small detail in my suggestion. The implementation I proposed was an expression-bodied getter-only property:
```cs
private static List<string> loadErrors = new();
public static IReadOnlyList<string> LoadErrors => loadErrors; // Mind the arrow here
```
This will effectively compile into something like this:
```cs
private static List<string> loadErrors = new();
public static IReadOnlyList<string> LoadErrors  {
  get {
    return loadErrors;
  }
}
```

The current implementation allows changing the value of `LoadErrors` from outside the `DafnyLanguageServer` class which I'd like to avoid.